docs/dev/retool/workflow.md

```diff
@@ -1 +1,74 @@
 # Retool Workflow
+
+[Retool workflows](https://retool.com/products/workflows) are great for automated tasks that need to be run on intervals. For our project, we use Retool workflows to routinely sync our database with Legal Server every day. Additionally, we take advantage of Retool workflows to create function abstractions that chain REST APIs to create, edit, or delete listings (i.e. limited case assignments) via the Retool application.
+
+## Legal Server Sync
+
+To get data from Legal Server, we added the provided API link as a REST API resource.
+After fetching data from Legal Server using said resource, we processed it as follows:
+
+1. Filter to only open cases
+2. Search legal problem code for keywords (i.e. asylum)
+3. Determine time commitment by legal problems
+4. Upsert cases and relevant languages and relief codes to Supabase
+5. Delete old stored cases that do not match what's stored in Legal Server
+
```

Comment:
Could we add another section titled "Other Workflows" that lists the other workflows? It might also be helpful to add a brief description of what each workflow achieves, doesn't need to be as detailed as the LegalServer sync workflow above.

<img width="271" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/cc83c1cd-496d-484f-aeee-ee43cd57286b">